src/math/crypto/pedersen_hash.zig

```diff
@@ -0,0 +1,21 @@
+const std = @import("std");
+const Felt252 = @import("../fields/starknet.zig").Felt252;
+const expectEqual = std.testing.expectEqual;
+
+const starknet_crypto = @cImport(@cInclude("starknet_crypto.h"));
+
+pub fn pedersenHash(a: Felt252, b: Felt252) Felt252 {
+    // convert Felt252 to big endian byte array
+    var a_bytes = a.toBytesBe();
+    var b_bytes = b.toBytesBe();
+    var res = [_]u8{0} ** 32;
```

Comment:
32 is used multiple times over the PR can this be declared as a global constant somewhere?